gu-actix/src/pipe.rs

```diff
@@ -36,9 +37,10 @@ impl<T, E> Stream for AsyncReader<T, E> {
 impl io::Read for SyncReader<Bytes, io::Error> {
     fn read(&mut self, buf: &mut [u8]) -> Result<usize, io::Error> {
         if self.buffer.is_none() {
-            let is_full = self.rx.is_full();
+            let is_full = self.rx.as_ref().unwrap().is_full();
```

Comment:
I had removed this y-day after your review for cleanup... it seems not to have any side effects. so pls remove it also here

Reply:
i put comment there. about this.